service/awsconfig/v10/version_bundle.go

```diff
@@ -10,8 +10,8 @@ func VersionBundle() versionbundle.Bundle {
 	return versionbundle.Bundle{
 		Changelogs: []versionbundle.Changelog{
 			{
-				Component:   "component",
-				Description: "Add your change here",
+				Component:   "kubernetes",
```

Comment:
Nitpick but could you make the component `cloudconfig`? This matches previous changelogs.

Reply:
done